### ISSUE TITLE
The --scheduler flag uses an unholy amount of bandwidth

### ISSUE THREAD

**Issue by roddds** [pricingassistant/mrq#77]
Before:
![screenshot from 2015-03-23 18 59 48](https://cloud.githubusercontent.com/assets/516447/6791598/e4df5e0a-d18e-11e4-9e08-61f26258317f.png)

After:
![screenshot from 2015-03-23 19 00 13](https://cloud.githubusercontent.com/assets/516447/6791605/f307d03e-d18e-11e4-85e2-52eb3b8cfc0b.png)

I would love to help troubleshooting and fixing this, but I have no idea where to start.

**Comment by sylvinus** [comment]
Hi!

It looks like your config variable ```scheduler_interval``` is set to 1, which makes MRQ check each second for jobs to reschedule. Is that what you want?

Thanks!

**Comment by roddds** [comment]
Hi @sylvinus! It looks like that's not the issue unfortunately -- even when using the `--scheduler_interval` set to a high interval like 600, it still raises the upload and download rates. Please notice the command-line arguments on the first line and the network statistics on the bottom bar.

Thank you very much for your assistance!

![screenshot from 2015-03-24 02 28 11](https://cloud.githubusercontent.com/assets/516447/6796236/0bf2b250-d1ce-11e4-81ec-1bc978bea5e0.png)

**Comment by sylvinus** [comment]
@roddds could you use tcpdump to see what is going on the wire? My guess is that if your measurement is real, it is caused by the cleaning jobs, not by the scheduler itself. It would be helpful to see if it is MongoDB, Redis or another kind of traffic.

Best,

**Comment by roddds** [comment]
@sylvinus I finally think I found out what happened. I had this machine running jobs for a few days, but the worker didn't have the scheduler enabled. When I finally started it, it probably had a huge backlog of jobs scheduled in the past, and then it had to go through every one of them to clean them up; your guess was correct.

Thank you very much for your support, I guess this closes it. I'll try sending a PR with a note about it to the docs.
